lightning-background-processor/src/lib.rs

```diff
@@ -36,11 +36,18 @@ use lightning::onion_message::messenger::AOnionMessenger;
 use lightning::routing::gossip::{NetworkGraph, P2PGossipSync};
 use lightning::routing::scoring::{ScoreUpdate, WriteableScore};
 use lightning::routing::utxo::UtxoLookup;
-use lightning::sign::{ChangeDestinationSource, OutputSpender};
+#[cfg(feature = "futures")]
```

Comment:
With the sweeper being natively async now, is there still much use for this feature flag?

Reply:
Will do in follow up